Street Trees/STConfirmationPageViewController.swift

```diff
@@ -0,0 +1,251 @@
+            let aState = address?.state ?? "No state specified"
+            let aZip = address?.state ?? "No zip specified"
+            textLabel = "\(aCity), \(aState) \(String(aZip))"
+            
+        case .UserName:
+            textLabel = contact?.name ?? "Error retrieving name"
+            
+        case .Phone:
+            textLabel = contact?.phoneNumber ?? "Error retrieving phone number"
+            
+        case .Email:
+            textLabel = contact?.email ?? "Error retrieving email"
+            
+        }
+        
+        cell.textLabel?.text = textLabel
+       
+        return cell
+    }
+    
+    func tableView(tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat
+    {
+        return 80.0
+    }
+    
+    func tableView(tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat
+    {
+        return 0.01
+    }
+    
+    func tableView(tableView: UITableView, titleForHeaderInSection section: Int) -> String?
+    {
+        return self.dataSource.item(forIndexPath: NSIndexPath(forRow: 0, inSection: section)).sectionHeader()
+    }
+    
+    func tableView(tableView: UITableView, viewForFooterInSection section: Int) -> UIView?
+    {
+        return UIView(frame: CGRectZero)
+    }
+    
+    func tableView(tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int)
+    {
+        let aHeader:UITableViewHeaderFooterView = view as! UITableViewHeaderFooterView
+        
+        aHeader.textLabel?.backgroundColor = UIColor.clearColor()
+        aHeader.textLabel?.font = UIFont.systemFontOfSize(22)
+        aHeader.textLabel?.textColor = UIColor.lightGrayColor()
+        aHeader.textLabel?.text? = aHeader.textLabel?.text?.capitalizedString ?? ""
+        aHeader.tintColor = UIColor.clearColor()
+    }
+    
+    
+    
+//**********************************************************************************************************************
+// MARK: - Actions:
+    
+    @IBAction func confirmButtonItemTouchUpInside(sender: AnyObject)
+    {
+        
+        if self.confirm()
+        {
+            
+
+            let alertConfimationMessage = UIAlertController(title: "Confirmation\n\n\n\n", message: "Thank You! Your request has been sent.", preferredStyle: .Alert)
+            
+            
+            let sentImage = UIImage(named: "sent")!.imageWithRenderingMode(UIImageRenderingMode.AlwaysOriginal)
+            let imageWidth = sentImage.size.width * 0.6
+            let imageHeight = sentImage.size.height * 0.6
+            let marginLeft:CGFloat = (alertConfimationMessage.view.bounds.size.width * 0.5) - (imageWidth * 0.72)
+            let marginTop:CGFloat = 42.0
+            let rect = CGRectMake(marginLeft, marginTop, imageWidth, imageHeight)
+            let imageView = UIImageView(frame: rect)
+            imageView.image = sentImage
+            alertConfimationMessage.view.addSubview(imageView)
```

Comment:
The image and all of this custom framing doesn't look good on the alert. If you want to use the confirmation image, I would create a new view controller and push that on, or remove the image all together.

Second reason to not use an image in an alert view is that you will need to create custom logic for every screen size as it varies from 4S to iPad Pro.